src/main/scala/uk/gov/hmrc/play/http/HeaderCarrier.scala

```diff
@@ -33,12 +34,14 @@ case class HeaderCarrier(authorization: Option[Authorization] = None,
                          requestId: Option[RequestId] = None,
                          requestChain: RequestChain = RequestChain.init,
                          nsStamp: Long = System.nanoTime(),
-                         extraHeaders: Seq[(String, String)] = Seq(), 
+                         extraHeaders: Seq[(String, String)] = Seq(),
```

Comment:
Hmmm tricky. I'd like to clarify the name 'extraHeaders' but I'm wary of introducing a breaking change. The only other thing I can think of is to rename 'otherHeaders' to something like 'headersNotPresentInBlacklist' which is a bit wordy